cmd/process-agent/app/events.go

```diff
@@ -64,13 +81,26 @@ func runEventListener(cmd *cobra.Command, args []string) error {
 		return log.Criticalf("Error parsing config: %s", err)
 	}
 
+	return nil
+}
+
+func printEvent(e *model.ProcessEvent) {
+	b, err := json.MarshalIndent(e, "", "  ")
+	if err != nil {
+		log.Error("Error while marshalling process event: ", err.Error())
```

Comment:
nit: You can use `Errorf` instead

Reply:
Thanks! Done in https://github.com/DataDog/datadog-agent/pull/12159/commits/5f95d06ed2feb6fecd77eb0f7c7086893b3f5b50.